ucum-java8/src/main/java/systems/uom/ucum/format/UCUMFormat.java

```diff
@@ -207,24 +207,48 @@ public Appendable format(Unit<?> unknownUnit, Appendable appendable)
 		    temp.append(')');
 		}
 		int pow = productUnits.get(u);
-		if (app.length() > 0) { // Not the first unit.
-		    if (pow >= 0) {
-			app.append('.');
-		    } else {
-			app.append('/');
-			pow = -pow;
-		    }
+				int indexToAppend;
+				if(app.length() > 0) { // Not the first unit.
+
+					if(pow >= 0) {
+
+						if(app.indexOf("1/") >= 0) {
+							indexToAppend = app.indexOf("1/");
```

Comment:
he wouldn't let me use final because I have a `indexToAppend++` in the code, and I calculated the index because it was better than use duplicated `app.insert(temp)` all over the code.